src/api/SFA.DAS.Provider.Events.Api.Client/PaymentsEventsApiClient.cs

```diff
@@ -43,6 +43,12 @@ private string BaseUrl
             return JsonConvert.DeserializeObject<PageOfResults<Payment>>(response);
         }
 
+        public async Task<PaymentStatistics> GetPaymentStatisctics()

```

Comment:
is it worth correcting the spelling typo here?

Reply:
yep, all done